docs/sources/developers/plugins/custom-panel-option-editors.md

```diff
@@ -117,4 +121,4 @@ export const SimpleEditor = ({ item, value, onChange, context }: StandardEditorP
 };
 ```
 
-Have you built a custom editor that you think would be useful to other plugin developers? Consider contributing it as a standard editor!
+Have you built a custom editor that you think would be useful to other plugin developers? Consider [contributing it](https://grafana.com/docs/grafana/latest/developers/contribute/) to have it considered as a standard editor.
```

Comment:
follow up: a link would help here to the standard editors or how one would be contribued

Reply:
There's already a link to contribution guidelines. Do you have a specific link to suggest to add or replace it? (I'm not sure how linking to standard editors would help since this is about custom editors.)

Reply:
It's suggesting that the custom editor the developer creates could instead be contributed as a standard editor. But having chatted with the team instead I think we can just remove this entire sentence.

Reply:
Deleted.